2025/build-tooling/demo/final/src/main.tsx

```diff
@@ -16,12 +16,20 @@ const placesServiceInfo = {
       : undefined,
 };
 
-ReactDOM.createRoot(document.getElementById("root")!).render(
+const root = document.getElementById("root");
+if (!root) {
+  throw new Error("No root element found");
+}
```

Comment:
I think we can skip this check - React throws a reasonably helpful error, it's non-conventional at least in tutorials, and keeps things short

Reply:
I remembered the other reason – while raged will throw an error since I on the court to be quite similar in non-demo two I wanted to include a manual check in both places